cmd/bank-vaults/configure.go

```diff
@@ -195,10 +203,11 @@ func watchConfigurations(parser multiparser.Parser, vaultConfigFiles []string, c
 
 		configFileDirs[configDirTrimmed] = files
 
-		logrus.Infof("watching directory for changes: %s", configDir)
+		slog.Info(fmt.Sprintf("watching directory for changes: %s", configDir))
 		err := watcher.Add(configDir)
 		if err != nil {
-			logrus.Fatal(err)
+			slog.Error(err.Error())
+			runtime.Goexit()
```

Comment:
I think we can manually close watcher before exiting rather than using `runtime.Goexit`. 
```suggestion
			// close watcher manually to avoid leaking memory on force exits
			_ = watcher.Close()
			os.Exit(1)
```